src/app/forms/custom-sections/letters/letters.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output } from '@angular/core';
+import { DynamicFormService } from '@forms/services/dynamic-form.service';
+import { CustomFormArray, CustomFormGroup, FormNode, FormNodeEditTypes } from '@forms/services/dynamic-form.types';
+import { LettersTemplate } from '@forms/templates/general/letters.template';
+import { Roles } from '@models/roles.enum';
+import { Axle, Letters, TechRecordModel, VehicleTypes } from '@models/vehicle-tech-record.model';
+import { debounceTime, Subscription } from 'rxjs';
+import { DocumentRetrievalService } from '@api/document-retrieval';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'app-letters',
+  templateUrl: './letters.component.html',
+  styleUrls: ['./letters.component.scss']
+})
+export class LettersComponent implements OnInit, OnDestroy, OnChanges {
+  @Input() vehicleTechRecord!: TechRecordModel;
+  @Input() isEditing = false;
+
+  @Output() formChange = new EventEmitter();
+
+  public form!: CustomFormGroup;
+  private _formSubscription = new Subscription();
+  public isError: boolean = false;
+  public errorMessage?: string;
+
+  constructor(
+    public dfs: DynamicFormService,
```

Comment:
No need for this to be public.

Reply:
The code started throwing a fit if it was set to something besides public, so I'm probably going to leave it be

Reply:
Your `form` should be public because it's accessed by the summary, but setting this to private shouldn't be an issue. It's a small thing, so it's fine for now.